src/main/java/org/openstreetmap/atlas/checks/maproulette/data/ProjectConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package org.openstreetmap.atlas.checks.maproulette.data;
+
+import java.io.Serializable;
+
+/**
+ * Helper class to decouple MapRouletteConfiguration from configuring projects. This allows easier
+ * updating of code that handles projects without touching general Map Roulette configuration code.
+ *
+ * @author nachtm
+ */
+public class ProjectConfiguration implements Serializable
+{
+    private String name;
```

Comment:
I would make all these variables final, as it seems that they are only set in the constructor and only accessed through getters.